packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -66,6 +66,8 @@ export type ChatCompositeOptions = {
 /**
  * A customizable UI composite for the chat experience.
  *
+ * @remarks Chat composite min width and height are respectively 19.5rem and 20rem (312px and 320px, with default rem at 16px)
```

Comment:
what does this `@remarks` doc do?

Reply:
it creates a "Remarks" field like this one:
![image](https://user-images.githubusercontent.com/82416644/141521019-7d3fbfd4-cce8-40c8-b974-89ca75c198ea.png)